### DISCUSSION TITLE
I have a question about a repository on GitHub

### DISCUSSION THREAD

**Discussion by AliAkbarnasiri** [community/community#109613] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a question about a repository on GitHub

**Comment by ettaboyle** [comment]
The GitHub Community Discussions is a space for all GitHub users to come and discuss ideas and projects. This post was flagged and locked because it violated our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way.

As this is a second offense, the original poster has been blocked indefinitely from this org.
